File: java/src/main/java/demo/officeDay/defaultMethod/CarInfo.java
package demo.officeDay.defaultMethod;

public interface CarInfo {

    String UNKNOWN_MAKER = "Unknown Maker";
    String UNKNOWN_MODEL = "Unknown Model";

    default String getMaker(){
        return UNKNOWN_MAKER;
    }

    default String getModel(){
        return UNKNOWN_MODEL;
    }

    int getMaxSpeed();

    static CarInfo create(int maxSpeed){
        return () -> maxSpeed;
    }
}

File: csharp/demo.officeDay/defaultMethod/CarInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace demo.officeDay.defaultMethod
{
    public interface CarInfo
    {
        string Model { get; }

        string Maker { get; }
  

        int getMaxSpeed();
    }
}
